### PR TITLE
Add Dropdown unit test

### PR DESCRIPTION
**Pull request by KieSun**
KxGMUh

### CODE REVIEW

File: src/dropdown/tests/Dropdown.spec.ts [hunk 1]
```diff
@@ -1,5 +1,78 @@
 import { mount } from '@vue/test-utils'
+import { ComponentPublicInstance, h, nextTick } from 'vue'
+import { NIcon } from '../../icon'
+import { CashOutline as CashIcon } from '@vicons/ionicons5'
 import { NDropdown } from '../index'
+import { VueWrapper } from '@vue/test-utils/dist/vueWrapper'
```
Comment (07akioni) [inline src/dropdown/tests/Dropdown.spec.ts:6]: import it at line 1

File: src/dropdown/tests/Dropdown.spec.ts [hunk 2]
```diff
@@ -1,5 +1,78 @@
...
+    children: [
+      {
+        label: '乔丹·贝克',
+        key: 'jordan baker'
+      }
+    ]
+  }
+]
+
+interface IProps {
```
Comment (07akioni) [inline src/dropdown/tests/Dropdown.spec.ts:52]: `DropdownProps` is exported. Please use that type.

File: src/dropdown/tests/Dropdown.spec.ts [hunk 3]
```diff
@@ -1,5 +1,78 @@
...
+  onSelect?: (key: string | number) => void
+  inverted?: boolean
+  data?: any[]
+}
+
+const render = ({
```
Comment (07akioni) [inline src/dropdown/tests/Dropdown.spec.ts:58]:
```suggestion
const mountDropdown = ({
```

File: src/dropdown/tests/Dropdown.spec.ts [hunk 4]
```diff
@@ -10,32 +83,125 @@ describe('n-dropdown', () => {
     })
   })
 
-  it('dropdown disabled', async () => {
+  it('show item', async () => {
```
Comment (07akioni) [inline src/dropdown/tests/Dropdown.spec.ts:86]:
```suggestion
  it('shows menu after click', async () => {
```

File: src/dropdown/tests/Dropdown.spec.ts [hunk 5]
```diff
@@ -1,5 +1,78 @@
 import { mount } from '@vue/test-utils'
+import { ComponentPublicInstance, h, nextTick } from 'vue'
+import { NIcon } from '../../icon'
+import { CashOutline as CashIcon } from '@vicons/ionicons5'
 import { NDropdown } from '../index'
+import { VueWrapper } from '@vue/test-utils/dist/vueWrapper'
+
+const className = 'n-dropdown-option-body n-dropdown-option-body--pending'
```
Comment (07akioni) [inline src/dropdown/tests/Dropdown.spec.ts:8]:
```suggestion
const pendingOptionClassName = 'n-dropdown-option-body n-dropdown-option-body--pending'
```

File: src/dropdown/tests/Dropdown.spec.ts [hunk 6]
```diff
@@ -1,5 +1,78 @@
...
+const selectors = '.n-dropdown-option-body'
```
Comment (07akioni) [inline src/dropdown/tests/Dropdown.spec.ts:9]:
```suggestion
const optionBodySelector = '.n-dropdown-option-body'
```

**Comment by KieSun** [comment]
all done

**Pull request merged**